### PR TITLE
Create Account and Send Money - request status / error handling, reusable components for account forms, AccountFormAccountId component for Recover Account

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

File: src/utils/wallet.js [hunk 1]
```diff
@@ -132,32 +132,50 @@ export class Wallet {
...
    }
 
-   async checkAccount(accountId) {
-      if (accountId !== this.accountId) {
-         return await this.near.nearClient.viewAccount(accountId)
-      } else {
-         throw new Error('You are logged into account ' + accountId + ' .')
+   async checkAccountAvailable(accountId) {
+      if (!this.isLegitAccountId(accountId)) {
+         throw new Error()
```
Comment (vgrichina) [inline src/utils/wallet.js]: let's not ever throw errors without meaningful messages. It makes life harder for everyone trying to debug.

File: src/utils/wallet.js [hunk 2]
```diff
@@ -132,32 +132,50 @@ export class Wallet {
...
+      }
+      if (accountId === this.accountId) {
+         throw new Error()
+      }
+      let remoteAccount = null
+      try {
+         remoteAccount = await this.near.nearClient.viewAccount(accountId)
+      } catch (e) {
+         throw new Error()
```
Comment (vgrichina) [inline src/utils/wallet.js]: what. is the point of rethrowing new Error here? It does nothing except hiding actual reason of error.

File: src/utils/wallet.js (hunk 1 above)
Comment (marcinbodnar) [reply src/utils/wallet.js]: fixed

File: src/utils/wallet.js (hunk 2 above)
Comment (marcinbodnar) [reply src/utils/wallet.js]: fixed as well

**Pull request closed without merging**